lib/python/rose/loc_handlers/rsync.py

```diff
@@ -61,33 +61,33 @@ def parse(self, loc, _):
         # Attempt to obtain the checksum(s) via "ssh"
         host, path = loc.name.split(":", 1)
         cmd = self.manager.popen.get_cmd(
-            "ssh", host, "python2", "-", path, loc.TYPE_BLOB, loc.TYPE_TREE)
+            "ssh", host, "python3", "-", path, loc.TYPE_BLOB, loc.TYPE_TREE)
         temp_file = TemporaryFile()
-        temp_file.write(r"""
```

Comment:
FYI `r` and `b` are not mutually exclusive:
```python
rb'foo\n'
```

Reply:
:+1: